lib/postcss-plugin.js

```diff
@@ -151,7 +151,7 @@ function replaceIconFontDeclarations (fontName, postCssRoot, svgPaths) {
  */
 function addFontDeclaration (fontName, postCssRoot, enforcedSvgHeight, svgPaths) {
   // The options are passed as a query string so we use the relative svg paths to reduce the path length per file
-  const options = { svgs: svgPaths.relative, name: fontName, enforcedSvgHeight: enforcedSvgHeight };
+  const options = { svgs: svgPaths.resolved, name: fontName, enforcedSvgHeight: enforcedSvgHeight };
```

Comment:
we should use only relative paths.. otherwise webpack won't be able to share caches